nusight2/src/client/components/localisation/robot_model.ts

```diff
@@ -168,18 +168,17 @@ export class LocalisationRobotModel {
   }
 
   /* Field to torso rotation in field space. */
```

Comment:
Describing it as "in a space" is kinda misleading anyway

```suggestion
  /* Torso to field rotation. */
```